common/djangoapps/student/models.py

```diff
@@ -1442,6 +1442,28 @@ def enrollments_for_user(cls, user):
         return cls.objects.filter(user=user, is_active=1)
 
     @classmethod
+    def preloaded_enrollments_for_user(cls, user):
+        """
```

Comment:
I also was going to add that I don't love the new name `preloaded_enrollments_for_user`, because I thought it was just going return the ones that were preloaded.  

Does the caller really need to think about whether or preload or not?  Is this performant enough that we could just leave this call as it was, and simply make the preloading a side effect documented in `enrollments_for_user`?